charts/gardener/provider-local/templates/rbac.yaml

```diff
@@ -129,7 +129,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ .Release.Namespace }}
+  name: {{ include "name" . }}-{{ .Release.Namespace }}
```

Comment:
Is `.Release.Namespace` not including a hash? If yes, how does this lead to clashes if deployed multiple times?

Reply:
The `.Release.Namespace` in the seeds does have a hash currently which comes from a random hash of a `ControllerInstallation` object. For any operator environment you don't really have this random hash, unless you generate one and save it in the extension resource status before any operation. 

Therefore for the case of a runtime cluster being a seed, then you have the seed extension running with a hash, but for the operator extension we can do w/e. In the hackathon we just used the extension name directly (which wouldn't have the hash)

Reply:
Thanks for the explanation.